pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -25,6 +25,20 @@ func (c *ContractReader) {{.Name}}({{range $index, $arg := .Arguments -}}
 		{{- range $arg := .Arguments -}}, {{.Name}}{{end}})
 	{{- end}}
 }
+{{- if eq .CallFlag "SessionIterator"}}
+
+// {{.Name}}Expanded is similar to {{.Name}} (uses the same contract
+// method), but can be useful if the server used doesn't support sessions and
+// doesn't expand iterators. It creates a script that will get the specified
+// number of result items from the iterator right in the VM and return them to
+// you. It's only limited by VM stack and GAS available for RPC invocations.
+func (c *ContractReader) {{.Name}}Expanded({{range $index, $arg := .Arguments -}}
+		{{- .Name}} {{.Type}},
+	{{- end}} _numOfIteratorItems int) ([]stackitem.Item, error) {
```

Comment:
It's not critical, but maybe we can handle this extra space before `_numOfIteratorItems` in case if there's no other arguments? `if len .Arguments` check will do the thing,

Reply:
Fixed.